sinonjs/formatio
# Migrate to CircleCI and Codecov

This PR configures CircleCI and Codecov.

Upgrading from node 4 and 6 will allow us to upgrade Mocha to 6.x, which dropped support for node 6.

#### How to verify - mandatory

1. Observe that verifications are run in CircleCI in node 8, 10 and 12
1. Observe that coverage is uploaded to Codecov
